packages/portfolio-contract/src/portfolio.contract.ts

```diff
@@ -80,7 +81,7 @@ export const AxelarConfigShape: TypedPattern<AxelarConfig> = M.splitRecord(
 );
 
 type Beefy = {
```

Comment:
For a type called `Beefy` I would expect it to be the whole ABI or something.

I suppose this declaration isn't exported, so it's not critical, but I suggest


```suggestion
type BeefyContracts = {
```